### PR TITLE
GEOMESA-1075 remove install vecmath script

### PR DESCRIPTION
**Pull request by jahhulbert-ccri**
Signed-off-by: Andrew Hulbert <andrew.hulbert@ccri.com>'

because jim said so!

### CODE REVIEW

**Comment by elahrvivaz** [comment]
should also modify `geomesa-tools/src/main/assemblies/component.xml`. Might just change that to be include * so that we don't have to worry about it anymore

File: geomesa-tools/src/main/assemblies/component.xml [hunk 1]
```diff
@@ -24,16 +24,7 @@
...
-                <include>geomesa</include>
-                <include>install-jai</include>
-                <include>install-jline</include>
-                <include>install-vecmath</include>
-                <include>install-imageio-ext</include>
-                <include>install-hadoop-accumulo.sh</include>
-                <include>install-kafka.sh</include>
-            </includes>
-            <filtered>true</filtered>
+            <filtered>false</filtered>
```
Comment (elahrvivaz) [inline geomesa-tools/src/main/assemblies/component.xml]: don't we need filtered true for e.g. https://github.com/locationtech/geomesa/commit/885d5b60ea5832b1a2a902c3f57d2e0b60e9d8f6#diff-051f96c3d5a44fe0603c870637be7e90R14 ?

**Comment by jahhulbert-ccri** [comment]
cool changed the component.xml

File: geomesa-tools/src/main/assemblies/component.xml (hunk 1 above)
Comment (jahhulbert-ccri) [inline geomesa-tools/src/main/assemblies/component.xml]: yeah :see_no_evil: i wasn't looking

**Comment by jahhulbert-ccri** [comment]
accidentally pushed to the wrong remote and it made travis mad when i removed it

**Comment by elahrvivaz** [comment]
haha, travis... ok looks good, merging

**Comment by elahrvivaz** [comment]
5a5748a5c59b30935e7d6443b0d5d049f118e388

**Pull request closed without merging**